src/common/Axios.ts

```diff
@@ -5,7 +5,7 @@ import axios from 'axios';
 
 export const AxiosConfig = axios.create({
     baseURL: process.env.REACT_APP_LAMBDA_URL,
-    timeout: 3000,
+    timeout: 5000,
```

Comment:
what is this based on can we document it? can we also document that this is in milli seconds

Reply:
The trends page often runs into a timeout when collecting commit data.

Will update this to add retry/show an error page as well.